package.json

```diff
@@ -70,7 +70,6 @@
     "@rollup/plugin-babel": "^6.0.3",
     "@rollup/plugin-node-resolve": "^15.1.0",
     "@testing-library/dom": "^10.1.0",
-    "@testing-library/jest-dom": "^6.2.0",
```

Comment:
This was unused, looks like the browser integration implements the same features.